get_started/simple_pipeline.md

```diff
@@ -116,7 +118,9 @@ defmodule Hello do
 end
 ```
 
-The main purpose of `handle_init` callback is to prepare our pipeline. Preparing among others means that we need to specify all its elements as children and set-up linkings between them. All other options that can be specified if needed you can find in `Membrane.ParentSpec` [docs](https://hexdocs.pm/membrane_core/Membrane.ParentSpec.html).
+The main purpose of the `handle_init` callback is to prepare our pipeline. Preparing means that we need to specify all its elements as children and set up links between those children to describe the order in which data will flow through the pipeline.
+Pipeline's callbacks are expected to return a status and an optional list of actions to be taken. The action can be of one of the following [types](https://hexdocs.pm/membrane_core/Membrane.Pipeline.Action.html#t:t/0).
```

Comment:
```suggestion
Pipeline's callbacks are expected to return status, an optional list of actions to be taken, and an updated state in which the pipeline will be after that callback. The action can be of one of the following [types](https://hexdocs.pm/membrane_core/Membrane.Pipeline.Action.html#t:t/0).
```

While suggesting previously, I have forgotten that the callback should also return an updated state - probably it is worth mentioning, just not to cause any misunderstandings.